github_actions/spec/dependabot/github_actions/update_checker_spec.rb

```diff
@@ -165,11 +165,11 @@
 
     context "given a dependency with a tag reference" do
       let(:reference) { "v1.0.1" }
-      it { is_expected.to eq("5273d0df9c603edc4284ac8402cf650b4f1f6686") }
+      it { is_expected.to eq(Gem::Version.new("1.1.0")) }
```

Comment:
Is it intentional to use `Gem::Version` here vs `GithubActions::Version`?

Reply:
Nope, it came from the exception:
```
     Failure/Error: it { is_expected.to eq("some hash i replaced") }

       expected: "some hash i replaced"
            got: #<Dependabot::GithubActions::Version "1.1.0">

       (compared using ==)

       Diff:
       @@ -1 +1 @@
       -"some hash i replaced"
       +Gem::Version.new("1.1.0")
```

I was lazy enough to copy/paste the logged string :)
I dig `Dependabot::GithubActions::Version` for clarity, switching.